woocommerce/woocommerce-gutenberg-products-block
# Start using the `block.json` metadata file for `Products by Tag`

The goal of this PR is to use the `block.json` metadata file to register the `Products by Tag` block.

### Testing

How to test the changes in this Pull Request:
1. Create a page and add a `Products by Tag` block.
2. Save the page and check it renders correctly the default config in the frontend.
3. Edit the page again and make some changes to the block (hiding some content, changing columns number, ordering, etc.).
4. Save the page and check it renders properly with the new config.
